### PR TITLE
floating Buttons added 

### PR DESCRIPTION
**Pull request by vaibhav18matere**
floating button

### CODE REVIEW

**Review by shivanigangadharan — COMMENTED** [review]
Code looks neat, good work!
Just 1 question, as these are floating buttons and should 'float above UI', can you use position:fixed to make them float like at the bottom right corner of the screen?

File: components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    border-radius: 5px;
+    border: none;
+    letter-spacing: 1.5px;
+    font-weight: 700;
+    color: var(--white);
+    cursor: pointer;
+    text-decoration: none;
+    line-height: 1.2rem;
+}
+.btn-primary{
```
Comment (junaidshaikh-js) [inline components/button/button.css:13]: There should be a blank line between each block of CSS styles. Use code formatter.

File: components/button/button.css [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+}
+.btn-link:hover{
+    background-color: var(--warning);
+}
+/* Outline buttons */
+.btn-outline{
+    padding: 0.8rem;
+    margin: 1.5rem;
+    cursor: pointer;
+    border-radius: 5px;
```
Comment (junaidshaikh-js) [inline components/button/button.css:83]: Code is repeating multiple times, better to use a single class or same block of code with multiple selectors.

File: components/button/button.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- font awesome link -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="/styles.css">
+    <link rel="stylesheet" href="button.css">
+    <title>Falcon UI | Home</title>
```
Comment (junaidshaikh-js) [inline components/button/button.html:11]: Title should match the content of page.

**Review by junaidshaikh-js — COMMENTED** [review]
Great Work!

**Pull request merged**